Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/remesh.h

```diff
@@ -41,9 +41,11 @@ namespace Polygon_mesh_processing {
 *         The descriptor types `boost::graph_traits<PolygonMesh>::%face_descriptor`
 *         and `boost::graph_traits<PolygonMesh>::%halfedge_descriptor` must be
 *         models of `Hashable`.
-*         If `PolygonMesh` has an internal property map for `CGAL::face_index_t`,
-*         and no `face_index_map` is given
-*         as a named parameter, then the internal one must be initialized
+*         If `PolygonMesh`
+  *  has an internal not writable property map
+  *  for `CGAL::face_index_t` and no `face_index_map` is given
+  *  as a named parameter, then the internal one must be initialized; else, it will be.
```

Comment:
```suggestion
*         If `PolygonMesh` has a non modifiable internal property map
*         for `CGAL::face_index_t` and no `face_index_map` is given
*         as a named parameter, then the internal one must be initialized; otherwise, it will be.
```